### PR TITLE
Version 0.2.2

### PR DESCRIPTION
**Pull request by SantiagoTorres**

### CODE REVIEW

**Comment by SantiagoTorres** [comment]
@lukpueh @vladimir-v-diaz pong. Could I get some of eyeballs on this? :beer: 

```
(in-toto) [santiago@LykOS in-toto]$ git log --merges --oneline 0.2.1..HEAD
faf64e0 (refs/remotes/origin/develop, refs/remotes/origin/HEAD) Merge pull request #199 from lukpueh/add-base-path-arg
26394c7 Merge pull request #201 from SantiagoTorres/fix_gpg_keyid_subkeys
1937bec Merge pull request #200 from SantiagoTorres/remove_pinned_version
b8b61ac (refs/heads/add_gpg_subkey_support) Merge pull request #198 from lukpueh/add-exclude-args
3244e1a Merge pull request #190 from lukpueh/gpg-subkeys
47685a9 Merge pull request #195 from lukpueh/normalize-path-prefix
6ff1e16 Merge pull request #192 from lukpueh/record-follow-links
e855f1d Merge pull request #197 from lukpueh/update-securesystemslib-0.10.10
05f2848 Merge pull request #191 from lukpueh/fix-exception-typo
e53abb1 Merge pull request #189 from lukpueh/sublayout-namespacing
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## in-toto v0.2.2
+
+* Add support for gpg signing subkeys.
+* Drop strict requirement on securesystemslib 0.9.
```
Comment (vladimir-v-diaz) [inline CHANGELOG.md:6]: Should it be 0.10.10?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+* Command line tool changes:
+    - Add a --base-path parameter to in-toto record and in-toto run
+    - in-toto-record now follows symbolic links
+* Fixed typo in exception messages
+* Adds support for sublayout namespacing (for in-toto spec 0.9 compliance)
+* Path prefix is normalized during in-toto verification:
+    - Paths such as foo//bar match with foo/bar.
```
Comment (vladimir-v-diaz) [inline CHANGELOG.md:13]: Should you mention this change?
https://github.com/in-toto/in-toto/commit/7ad8942b6c9082f8e5773f8fb0f2ed3375b86f18

Comment (SantiagoTorres) [reply CHANGELOG.md:13]: Thanks for the heads up! will add

**Comment by SantiagoTorres** [comment]
@vladimir-v-diaz updated. Thanks!

**Comment by vladimir-v-diaz** [comment]
You can go ahead and merge this, I didn't notice any other issues.

**Pull request merged**